.github/workflows/terrascan.yaml

```diff
@@ -0,0 +1,24 @@
+name: Terrascan
+
+on:
+  push:
+#    paths:
+#     - 'chart/k8gb/**'
+
+jobs:
+  terrascan:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Run Terrascan
+      id: terrascan
+      uses: accurics/terrascan-action@main
```

Comment:
Is `@main` branch desired ?

Reply:
@kuritka just grabbed it from the example at https://github.com/accurics/terrascan-action, still testing